frontend/src/app/shared-components/toolbar/toolbar.component.html

```diff
@@ -1,7 +1,7 @@
 <mat-toolbar color="primary" fxLayout="row">
   <div class="left" fxFlex="100px">
     <a *ngIf="backUrl" [routerLink]="backUrl" class="clickable">
-      <mat-icon>keyboard_arrow_left</mat-icon>
+      <mat-icon class="icon-arrow">keyboard_arrow_left</mat-icon>
```

Comment:
**issue:** `mat-icon` should have a property called color: https://material.angular.io/components/icon/api#MatIcon